benchmarks/samples/fabric/fabcar/queryAllCars.js

```diff
@@ -0,0 +1,44 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+module.exports.info = 'Querying all cars.';
+let startIndex = 1;
+let endIndex = 50;
+let bc, contx;
+
+module.exports.init = function (blockchain, context, args) {
+    bc = blockchain;
+    contx = context;
+
+    return Promise.resolve();
+};
+
+module.exports.run = function () {
+    let startingKey = 'CAR' + startIndex.toString();
+    let endingKey = 'CAR' + endIndex.toString();
```

Comment:
start/end key would be handy variable to pass in from the test arguments in the config file, as a user might want to experiment with large/smaller ranges